lib/chat_api/messages/notification.ex

```diff
@@ -54,6 +54,22 @@ defmodule ChatApi.Messages.Notification do
     message
   end
 
+  def notify(%Message{private: false} = message, :sms, _opts) do
+    Logger.info("Sending message notification: :sms")
+
+    Task.start(fn ->
```

Comment:
Just wondering if if we should make this task [supervised](https://hexdocs.pm/elixir/master/Task.html#module-dynamically-supervised-tasks). So that we can make the node wait for pending tasks before it shuts down. [Task.start](https://hexdocs.pm/elixir/master/Task.html#start/1) wont wait

Reply:
yeah that could be a good idea! i've mostly just used `Task.start` in the meantime for simplicity, so if we can make this more robust with supervision that would be great :+1: